Sources/SCNPath/RKPathEntity.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  RKPathEntity.swift
+//  SCNPath+Example
+//
+//  Created by Grant Jarvis on 3/13/21.
+//  Copyright © 2021 Max Cobb. All rights reserved.
+//
+
+import SceneKit
+import Combine
+import RealityKit
+
+public class RealityKitPathEntity : SCNPathNode {
+    
+    ///The SceneKit scene that this node belongs to.
+    public weak var scene : SCNScene?
+    
+    ///A realityKit entity which represents the generated path.
+    public var realityKitPath = Entity()
+    
+    ///The number of times we have generated a .usdz file
+    ///This is used for changing the file path each time.
+    private var entityCount = 0
+    
+    private var circleEntity : Entity?
+    
+    private var rectangleEntity : Entity?
+    
+    private var pathEntitiesInScene = [Entity]()
+    
+    override init(
+        path: [SCNVector3], width: Float = 0.5,
+        curvePoints: Float = 8, materials: [SCNMaterial] = []
+    ) {
+        super.init(path: path,
+                   width: width,
+                   curvePoints: curvePoints,
+                   materials: materials)
+        do {
+            let shapesScene = try ShapesProj.loadShapesScene()
```

Comment:
Yes, no problem. I can export from Reality Composer as a .usdz file and load the shapes in that way.

Reply:
I don't think it's necessary if the geometry is calculated from SceneKit though? is the issue just that you'd prefer it have a rounded end to the path? If so then it's probably something that should be added to the SceneKit geometry when constructed
I'd like to avoid adding non .swift files to the package if possible

Reply:
Yes, I totally agree. This is not apart of the .usdz conversion option (Option #1). I didn't mean to include Option #2 (generating the path in RealityKit with circles and rectangles) in this pull request. I was loading entities to round the path ends in RealityKit but I just realized I can do that with cornerRadius on MeshResource.generatePlane() - therefore my next commit will be a better version of Option #2 - with No loading of anything from a file, just generating the path in RealityKit programmatically. After commit that I will remove Option #2 entirely and just go back to converting from SceneKit.